### PR TITLE
CDPT-384 Amendments

### PR DESCRIPTION
**Pull request by njpreddy**
## Description
- Fixed the spacing issue between email addresses when multiple entries listed.
- Placed the 'No email address' message in an MoJ style banner rather than a gov.uk warning box.
- Reduced the space between the 'Send' button and 'Cancel' link.
 
## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
* [x] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [x] (8) Data migration script is created if any of letter templates is changed

### Screenshots

**With and email address present**

commissioning-with-email

**Without an email address present**

commissioning-without-email

### Related JIRA tickets

https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/1152?modal=detail&selectedIssue=CDPT-384

### Deployment

N/A
### Manual testing instructions

As per the scenarios in the Jira ticket.

### CODE REVIEW

**Review by lucas-shaw — APPROVED** [review]
LGTM

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -75,4 +75,4 @@
 @import "moj/case_tabs";
 @import "moj/definition-list";
 @import "moj/govuk_overrides";
-@import "moj/component/govuk_warning";
```
Comment (vertism) [inline app/assets/stylesheets/application.css.scss:78]: does moj/component/govuk_warning need to be removed?

File: app/assets/stylesheets/moj/_buttons.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
   }
 
   .button {
-    margin-right: $gutter;
+    margin-right: $gutter-half;
```
Comment (vertism) [inline app/assets/stylesheets/moj/_buttons.scss:28]: Does this affect any other buttons?

File: app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (njpreddy) [reply app/assets/stylesheets/application.css.scss:78]: govuk_warning component removed.

File: app/models/data_request.rb [hunk 3]
```diff
@@ -78,7 +78,7 @@ def request_dates_absent?
   end
 
   def recipient_emails
-    emails = contact&.data_request_emails&.split(%r/\s/) || []
+    emails = contact&.data_request_emails&.split(" ") || []
```
[the remark this answers is not in the document]
Comment (njpreddy) [reply app/models/data_request.rb:81]: Model test now added.

**Review by vertism — APPROVED** [review]
Lovely work!

**Pull request merged**